pootle/templates/browser/_actions.html

```diff
@@ -1,5 +1,6 @@
 {% load i18n %}
 <ul>
+  {% if not can_suggest_only %}
```

Comment:
> Did a test with a user with no rights and this link is displayed.

Made sure I don't have any rights being inherited from `default`, so the issue is still present.